source/server/http/admin.h

```diff
@@ -291,6 +294,8 @@ class AdminFilter : public Http::StreamDecoderFilter,
   const Http::StreamDecoderFilterCallbacks& getDecoderFilterCallbacks() const override;
   const Http::HeaderMap& getRequestHeaders() const override;
 
+  static void populateFallbackResponseHeaders(Http::Code code, Http::HeaderMap& header_map);
```

Comment:
Should this just be in the anonymous namespace in the `.cc`?

Reply:
yes, done.